docs/user_guide/tutorial.md

```diff
@@ -447,7 +448,7 @@ local_fraction_grid_for_EMEP_additional_grid_interpolation = 2
 
 First we define the type of wind data used for dispersion. 
 
-```
+```fortran
```

Comment:
It would be helpful with a "copy" button on the code blocks.

Reply:
I agree, but I don't think this is possible unfortunately